tests/test_requests.py

```diff
@@ -518,7 +518,8 @@ def test_basicauth_encodes_byte_strings(self):
         ))
     def test_errors(self, url, exception):
         with pytest.raises(exception):
-            requests.get(url, timeout=1)
+            r = requests.get(url, timeout=1)
+            assert r.exception == exception
```

Comment:
Well I was more concerned that you were beginning to attach exceptions to Response objects. You don't need this assertion. `with pytest.raises(exception)` is the assertion in this test.